docs/sphinx/examples/cpp/providers/nvqc_state.cpp

```diff
@@ -1,17 +1,17 @@
 // Compile and run with:
 // ```
-// nvq++ --target nvcf --nvcf-backend tensornet nvcf_state.cpp -o out.x
+// nvq++ --target nvqc --nvqc-backend tensornet nvqc_state.cpp -o out.x
 // ./out.x
 // ```
-// Assumes a valid NVCF API key and function ID have been stored in environment
-// variables or `~/.nvcf_config` file. Alternatively, they can be set in the
+// Assumes a valid NVQC API key and function ID have been stored in environment
+// variables or `~/.nvqc_config` file. Alternatively, they can be set in the
 // command line like below.
 // ```
-// nvq++ --target nvcf --nvcf-backend tensornet --nvcf-api-key <YOUR API KEY> \
-// --nvcf-function-id <NVCF function Id> nvcf_state.cpp -o out.x
+// nvq++ --target nvqc --nvqc-backend tensornet --nvqc-api-key <YOUR API KEY> \
+// --nvqc-function-id <NVQC function Id> nvqc_state.cpp -o out.x
```

Comment:
Same comment here, and same for the python examples.

Reply:
Fixed in https://github.com/NVIDIA/cuda-quantum/commit/ae21a9b4165762420178cb7cbfa3a4cdf89892af